qcodes_contrib_drivers/drivers/Keysight/SD_common/SD_AWG.py

```diff
@@ -14,9 +15,20 @@ class SD_AWG(SD_Module):
     This driver was written with the M3201A card in mind.
 
     This driver makes use of the Python library provided by Keysight as part of the SD1 Software package (v.2.01.00).
+
+    Args:
```

Comment:
Can we please remove the types from docstring and provide them in the corresponding function signatures? This comment applies to everywhere in your contribution so I will not repeatedly write the same thing.